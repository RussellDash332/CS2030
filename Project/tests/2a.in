2 1
0.500 1.000
0.600 1.000
0.700 1.000
1.500 1.000
1.600 1.000
1.700 1.000
^D