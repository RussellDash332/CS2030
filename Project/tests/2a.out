0.500 1 arrives
0.500 1 serves by server 1
0.600 2 arrives
0.600 2 serves by server 2
0.700 3 arrives
0.700 3 waits at server 1
1.500 1 done serving by server 1
1.500 3 serves by server 1
1.500 4 arrives
1.500 4 waits at server 1
1.600 2 done serving by server 2
1.600 5 arrives
1.600 5 serves by server 2
1.700 6 arrives
1.700 6 waits at server 2
2.500 3 done serving by server 1
2.500 4 serves by server 1
2.600 5 done serving by server 2
2.600 6 serves by server 2
3.500 4 done serving by server 1
3.600 6 done serving by server 2
[0.450 6 0]
